### DISCUSSION TITLE
Warning: You cannot give refs to function components. Attempts to access this reference will fail. Did you mean to use React.forwardRef()?

### DISCUSSION THREAD

**Discussion by arcoirisapp** [react-hook-form/react-hook-form#8453] [General]
Hello, we migrated from v6 to v7 but show :
**Warning: You cannot give refs to function components. Attempts to access this reference will fail. Did you mean to use React.forwardRef()?**

We use elastic ui , could you guide us ,react hoook form is a great library and thanks for all your support

https://codesandbox.io/s/app-elastic-crpj1e-crpj1e?file=/src/App.js

**Comment by bluebill1049** [comment]
This maybe an issue with that ui lib which ref is not forward, could you post a question there?

**Reply by arcoirisapp** [reply]
not yet but i will, thanks 👍
